### PR TITLE
@kanaabe => QA

### PR DESCRIPTION
**Pull request by eessex**
Opened in favor of https://github.com/artsy/reaction/pull/327

Addresses points in artsy/publishing#165, artsy/publishing#166, and artsy/publishing#168

### CODE REVIEW

**Comment by kanaabe** [comment]
Nice set of QA! 👍

File: src/Components/Artsy.tsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
 import PropTypes from "prop-types"
 import React from "react"
 
+// TODO: Once this PR for `rest` https://github.com/Microsoft/TypeScript/pull/13470 lands
+// we’ll be able to not make this optional and simply remove it from the props that a
+// component wrapped with the `ContextConsumer` HOC accepts.
+
```
Comment (alloy) [inline src/Components/Artsy.tsx]: @eessex Curious what the reason was for this, was it cluttering up the doc view in VSCode?

Comment (eessex) [reply src/Components/Artsy.tsx]: The todo was causing a type error I believe- but my memory is a bit foggy now

Comment (alloy) [reply src/Components/Artsy.tsx]: Oh weird. Ok, no problemo, just wondering 👍

**Pull request merged**